src/schema/SchemaGenerator.ts

```diff
@@ -47,9 +47,11 @@ export class SchemaGenerator {
 
     Object.keys(metadata.getModels()).forEach((modelName: string) => {
       const model: ModelMetadata = metadata.getModel(modelName);
+      const superName = model.klass ? model.klass.__proto__.name : null;
+      const superModel = superName ? metadata.getModel(superName) : undefined;
 
       template += `
-        ${entityToOrderByEnum(model)}
+        ${entityToOrderByEnum(model, superModel)}
```

Comment:
Any reason you don't just pass model in here as before and handle the super class logic in the `entityToOrderByEnum` function (just to maintain consistency here)?

Reply:
I moved it, as per comments